src/autolabel/tasks/classification.py

```diff
@@ -55,6 +60,40 @@ def construct_prompt(self, input: Dict, examples: List) -> str:
         # prepare task guideline
         labels_list = self.config.labels_list()
         num_labels = len(labels_list)
+
+        # if large number of labels, filter labels_list by similarity of labels to input
+        if num_labels >= 50:
+            example_prompt = PromptTemplate(
+                input_variables=["input"],
+                template="{input}",
+            )
+            label_examples = [{"input": label} for label in labels_list]
+
+            example_selector = SemanticSimilarityExampleSelector.from_examples(
+                # This is the list of labels available to select from.
+                label_examples,
+                # This is the embedding class used to produce embeddings which are used to measure semantic similarity.
+                OpenAIEmbeddings(),
+                # This is the VectorStore class that is used to store the embeddings and do a similarity search over.
+                VectorStoreWrapper(cache=False),
+                # This is the number of examples to produce.
+                k=10,
```

Comment:
We should have this value of `k` be configurable. Maybe 10 is a reasonable default, but we might want workflows in the future that automatically test for the right value of `k`.

Reply:
done in [f97a82d](https://github.com/refuel-ai/autolabel/pull/561/commits/f97a82d57b03bc3da5c59604b23ed7a573d230f7)

can now specify value of k in autolabel config file like so:
```
        "label_selection": true,
        "label_selection_count": 10
```